spec/models/institution_builder_spec.rb

```diff
@@ -21,18 +26,20 @@
         expect(described_class.run(user)[:success]).to be_truthy
       end
 
-      it 'returns the new preview version record if sucessful' do
-        create :version
-        old_version = Version.current_preview
-
-        version = described_class.run(user)[:version]
-
+      def expects_successful_preview(version, old_version)
         expect(version).to eq(Version.current_preview)
         expect(version).not_to eq(old_version)
         expect(version.production).to be_falsey
         expect(version).not_to be_generating
       end
 
+      it 'returns the new preview version record if sucessful' do
+        create :version
+        old_version = Version.current_preview
+        version = described_class.run(user)[:version]
+        expects_successful_preview(version, old_version)
```

Comment:
The overview of this cop is
> A long example is usually more difficult to understand. Consider extracting out some behaviour, e.g. with a `let` block, or a helper method.

I'm fine with extracting commonly used expectations to methods, but I think here this does not help make the example easier to understand.  Since this is the only spec that uses this method, I just have to go look in two different places to understand the whole spec.

I think this cop also assumes we are writing specs with one expectation each.  This is generally regarded as best practice.  Here we have 4 expectations.  However, breaking this up into four tests adds a lot of overhead (i.e. the objects would have to be created in the database 4 times), so I'm not necessarily a stickler for that "best practice".

I think we should move the expectations back into this spec and bump the max number of lines for this cop in `.rubocop.yml` to.....8?  That gives us room since we often have multiple expectations in a spec, yet still will let us know that a spec might be getting unwieldy. 
Thoughts?

Reply:
My team and I have discussed that changing the line limit to 8 will be the best move. I'm going to go back and refactor these files and then get back with you.